arcgis-ios-sdk-samples/Route & Directions/Find closest facility to an incident (interactive)/FindClosestFacilityInteractiveViewController.swift

```diff
@@ -0,0 +1,120 @@
+        AGSFacility(point: AGSPoint(x: -1.3040539754780494E7, y: 3862924.5938606677, spatialReference: .webMercator())),
+        AGSFacility(point: AGSPoint(x: -1.3042571225655518E7, y: 3858981.773018156, spatialReference: .webMercator())),
+        AGSFacility(point: AGSPoint(x: -1.3039784633928463E7, y: 3856692.5980474586, spatialReference: .webMercator())),
+        AGSFacility(point: AGSPoint(x: -1.3049023883956768E7, y: 3861993.789732541, spatialReference: .webMercator()))
+    ]
+    
+    // Create the graphics and add them to the graphics overlay
+    func makeFacilitiesOverlay() -> AGSGraphicsOverlay {
+        let facilitySymbolURL = URL(string: "https://static.arcgis.com/images/Symbols/SafetyHealth/Hospital.png")!
+        let facilitySymbol = AGSPictureMarkerSymbol(url: facilitySymbolURL)
+        facilitySymbol.height = 30
+        facilitySymbol.width = 30
+
+        let graphicsOverlay = AGSGraphicsOverlay()
+        graphicsOverlay.graphics.addObjects(from: facilities.map { AGSGraphic(geometry: $0.geometry, symbol: facilitySymbol) })
+        return graphicsOverlay
+    }
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        // Add the source code button item to the right of navigation bar.
+        (self.navigationItem.rightBarButtonItem as! SourceCodeBarButtonItem).filenames = ["FindClosestFacilityInteractiveViewController"]
+    }
+}
+
+// MARK: - AGSGeoViewTouchDelegate
+extension FindClosestFacilityInteractiveViewController: AGSGeoViewTouchDelegate {
+    func geoView(_ geoView: AGSGeoView, didTapAtScreenPoint screenPoint: CGPoint, mapPoint: AGSPoint) {
+        // Find the closest facilities with the default parameters.
+        closestFacilityTask.defaultClosestFacilityParameters { [weak self] (parameters, error) in
+            guard let self = self else { return }
+            if let parameters = parameters {
+                parameters.setFacilities(self.facilities)
+                
+                // Create a symbol for the incident.
+                let incidentSymbol = AGSSimpleMarkerSymbol(style: .cross, color: .black, size: 20)
+                
+                // Remove previous graphics.
+                self.incidentGraphicsOverlay.graphics.removeAllObjects()
+                
+                // Create a point and graphics for the incident.
+                let graphic = AGSGraphic(geometry: mapPoint, symbol: incidentSymbol)
+                self.incidentGraphicsOverlay.graphics.add(graphic)
+                
+                // Set the incident for the parameters.
+                parameters.setIncidents([AGSIncident(point: mapPoint)])
+                self.closestFacilityTask.solveClosestFacility(with: parameters) { [weak self] (result, error) in
+                    guard let self = self else { return }
+                    if let result = result {
+                        // Get the ranked list of colsest facilities.
```

Comment:
```suggestion
                        // Get the ranked list of closest facilities.
```
Fix typo in comment.